_additional/fakeClient/lib/ClientHandler.js

```diff
@@ -0,0 +1,139 @@
+const DataGenerator = require('./DataGenerator');
+const request = require('request-promise');
+
+const MESSAGE_DELAY_MIN = 1000;
+const MESSAGE_DELAY_MAX = 5000;
+
+class ClientHandler {
+	constructor(address) {
+		this._address = `http://${address}`;
+		this._isRunning = false;
+		// Next line is a trick used to print column-like messages in console.
+		// It's something like 'zero fill' with 23 characters length and spaces instead of zeros.
+		this._logPrefix = `[${address}]               `.substr(0, 23);
```

Comment:
logPrefixLength = 23
logPrefixBeginning = `[${address}]`
logPrefixFill = ' '.repeat(23 - logPrefixBeginning)
this._logPrefix = logPrefixBeginning + logPrefixFill
?

Reply:
U góry mi się nie podoba ta dziwna ilość spacji na końcu.

Reply:
Ech, taką fajną JSową sztuczkę rozbiłeś na pierdyliard linii kodu... Teraz trzeba to będzie wynieść do innego pliku (bo głupio trzymać logikę formatowania logów w ClientHandlerze).